sktalk/corpus/parsing/cha.py

```diff
@@ -12,22 +12,37 @@ def _extract_metadata(self):
         return self._pla_reader().headers()[0]
 
     def _extract_utterances(self):
-        reader = self._pla_reader().utterances(by_files=False)
-        return [self._to_utterance(read_utterance) for read_utterance in reader]
+        with open(self._path, encoding="utf-8") as f:
```

Comment:
regex for extracting lines (probably better to be placed on top of the class):

```python
    SPEAKER_REGEX = r"^\*(":P<name>[^\:])\:"
    TIME_REGEX = r"..."

    line_with_speaker = SPEAKER_REGEX + "\s+" + UTTERANCE_REGEX + TIME_REGEX
```
(TODO)